install.md

```diff
@@ -10,6 +10,32 @@ sudo pacman -S podman
 
 If you have problems when running Podman in [rootless](README.md#rootless) mode follow the instructions [here](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
 
+#### [Debian](https://debian.org)
+
+The [libpod](https://ftp-master.debian.org/new/libpod_1.6.4+dfsg-1.html) package is expected to be available
+soon in the default Debian repos.
```

Comment:
I'm not a fan of listing a particular version or saying "soon" in a doc like this.  Two months from now when v1.7 is well entrenched, customers will be annoyed with a pointer to 1.6.4 and "soon" will be irrelevant.

I'd prefer:

```
The libpod package is expected to be available in the by Spring 2020 in the default Debian repos.  Until that time, the [libpod v1.6.4](https://ftp-master.debian.org/new/libpod_1.6.4+dfsg-1.html) is available for download.

Reply:
ack thanks, I'll update

Reply:
@TomSweeneyRedHat updated the comment on official debian package. PTAL.

Reply:
I wasn't able to download binaries from the prior link, so I skipped it.